### ISSUE TITLE
Easier modding access for explosives and explosions

### ISSUE THREAD

**Issue by randomuranium235core** [studio-minus/ppg-bugs-and-requests#2265]
**Feature description**
Making explosions and explosives more customizable for modding. Such as changing explosion cloud textures, settings to the explosion (things like force and damage or shrapnel amount, etc.)

**Feature purpose**
This allows for a lot more customization for explosives for mods. May help modders create explosives that are more creative or have different ideas in mind rather than what we currently have.

**Additional context**
The current explosives in mods are very bland due to the limited amount of modding you can do with them. If we were able to customize the explosives further, we can (in a quite literal, and dummied down sense) can make more creative explosives that appeal more to players.

**Comment by blocker2225** [comment]
> **Feature description** Making explosions and explosives more customizable for modding. Such as changing explosion cloud textures, settings to the explosion (things like force and damage or shrapnel amount, etc.)
> 
> **Feature purpose** This allows for a lot more customization for explosives for mods. May help modders create explosives that are more creative or have different ideas in mind rather than what we currently have.
> 
> **Additional context** The current explosives in mods are very bland due to the limited amount of modding you can do with them. If we were able to customize the explosives further, we can (in a quite literal, and dummied down sense) can make more creative explosives that appeal more to players.

settings for explosions are already a thing

**Comment by DindinYT37** [comment]
Why did you quote the whole post?

**Comment by royalealbert27** [comment]
> > **Feature description** Making explosions and explosives more customizable for modding. Such as changing explosion cloud textures, settings to the explosion (things like force and damage or shrapnel amount, etc.)
> > **Feature purpose** This allows for a lot more customization for explosives for mods. May help modders create explosives that are more creative or have different ideas in mind rather than what we currently have.
> > **Additional context** The current explosives in mods are very bland due to the limited amount of modding you can do with them. If we were able to customize the explosives further, we can (in a quite literal, and dummied down sense) can make more creative explosives that appeal more to players.
> 
> settings for explosions are already a thing

It cannot make custom explosion clouds

**Comment by randomuranium235core** [comment]
> > **Feature description** Making explosions and explosives more customizable for modding. Such as changing explosion cloud textures, settings to the explosion (things like force and damage or shrapnel amount, etc.)
> > **Feature purpose** This allows for a lot more customization for explosives for mods. May help modders create explosives that are more creative or have different ideas in mind rather than what we currently have.
> > **Additional context** The current explosives in mods are very bland due to the limited amount of modding you can do with them. If we were able to customize the explosives further, we can (in a quite literal, and dummied down sense) can make more creative explosives that appeal more to players.
> 
> settings for explosions are already a thing

I don't think you understand what I meant or you just didn't read anything I mentioned.

**Comment by DindinYT37** [comment]
Duplicate of or Related to:
* #1460 
* #1498
* #2206
